src/test/e2e/01_component_choice_test.go

```diff
@@ -40,11 +38,13 @@ func TestComponentChoice(t *testing.T) {
 	require.NoFileExists(t, secondFile)
 
 	// Deploy using default choice
-	stdOut, stdErr, err = e2e.ExecZarfCommand("package", "deploy", path, "--confirm")
+	stdOut, stdErr, err = e2e.Zarf("package", "deploy", path, "--confirm")
 	require.NoError(t, err, stdOut, stdErr)
 
 	// Verify the file was created
 	require.FileExists(t, secondFile)
 
-	e2e.CleanFiles(firstFile, secondFile)
+	t.Cleanup(func() {
```

Comment:
Can we move this to the top of the test so that the files still get cleaned up on a failed test run <3